ansible_builder/main.py

```diff
@@ -119,7 +120,7 @@ def build_command(self):
         return command
 
     def build(self):
-        logger.debug(f'Ansible Builder is building your execution environment image, "{self.tag}".')
+        logger.debug('Ansible Builder is building your execution environment image. Tags: %s' % (', '.join(self.tags)))
```

Comment:
Why did you drop the format string? Not that concerned but curious:)

Reply:
Oops, I meant to go back and correct that before I pushed up the change. I'll correct it.